### ISSUE TITLE
Clarify or eliminate link between PARAT and CAT

### ISSUE THREAD

**Issue by za158** [georgetown-cset/parat#305]
https://github.com/georgetown-cset/eto-cat/issues/459

**Comment by za158** [comment]
@jmelot @rggelles Is the link between PARAT and CAT cited in this (outdated) documentation text still a thing? https://eto.tech/tool-docs/cat/#identifying-ai-related-companies

**Comment by rggelles** [comment]
Nope, I actually think when CAT was first made I suggested that relying on PARAT as a measure of AI-ness might not make sense, so happy to remove that reliance now.

**Comment by za158** [comment]
Great. @jmelot let me know when this is done (no rush) and I will update CAT documentation accordingly
